Gemfile

```diff
@@ -7,8 +7,6 @@ group :test, :development do
   gem 'pry'
   gem 'scc-codestyle', '0.1.4'
   gem 'ffi', '1.9.18'
-  gem 'guard'
-  gem 'guard-rspec', require: false
```

Comment:
@felixsch i had to remove guard, because it did not play well together with our other Gemfile requirements